docs/luigi-core-api.md

```diff
@@ -0,0 +1,192 @@
+# Luigi Core API
+
+## Luigi.config()
+
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
+
+### Configuration
+
+#### setConfig
+
+Sets the configuration for Luigi initially. Can also be called at a later point in time again to update the configuration.
+
+##### Parameters
+
+-   `configInput` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)?** the Luigi Core configuration object
```

Comment:
Please remove the question mark at the end (by removing the square brackets around the parameter name in the source file).